### ISSUE TITLE
Towards LogisticRegression with Skorch

### ISSUE THREAD

**Issue by michaelchughes** [tufts-ml/time_series_prediction#15]
I have added two files on the add_skorch branch:
* SkorchLogisticRegression.py
* SkorchLogisticRegressionModule.py
Found here: https://github.com/tufts-ml/time_series_prediction/tree/add_skorch/src/logistic/

These should be decent implementations of logistic regression using skorch framework.
@rathp can you try these out? Goal is that given the same inputs, both the sklearn (existing) and skorch (new) provide nearly identical classifiers.

## TODO list (use any simple dataset, like the EEG seizure/not seizure one):

* [ ] Look at a range of minibatch sizes and learning rates, select one/two that looks good
* [ ] Show a plot of loss function on training data, showing the loss vs epoch for skorch, and a dashed line for the loss of the final sklearn solution. We should see the lines converge to the same solution. Be careful to set the regularization term exactly the same and any class_weight exactly the same.
* [ ] Show the learned coefficients (bias and a weight for every feature).
